### DISCUSSION TITLE
Small datasheet error

### DISCUSSION THREAD

**Discussion by Sam-Vervaeck** [stnolting/neorv32#959] [General]
In the section of the linker script in the datasheet, the links on the right side of the screenshot should be swapped. I assume DMEM should be linked to in the ram row and IMEM in the rom row instead of how it is now?

![image](https://github.com/user-attachments/assets/a352b803-9bff-421f-b3e0-f15ba8ce5db5)

**Comment by stnolting** [comment]
You are right. Thanks for the hint! I'll fix that.

**Reply by Sam-Vervaeck** [reply]
To pick in on this thread I also want to ask about this section of the datasheet about the linker script:

![image](https://github.com/user-attachments/assets/3764755f-dc56-4812-b878-26191eb86454)

Should the parts in brackets not also be switched?

**Reply by stnolting** [reply]
Right. Looks like I was consistent in my swapping.. 😅

**Reply by stnolting** [reply]
Both typos are fixed now.

**Reply by Sam-Vervaeck** [reply]
> Right. Looks like I was consistent in my swapping.. 😅

![image](https://github.com/user-attachments/assets/942d6325-b912-4c32-904a-7770abc77d69)

No problem :)

**Reply by stnolting** [reply]
🤣
